bundles/CoreBundle/Migrations/Version20210323082921.php

```diff
@@ -57,7 +57,7 @@ public function up(Schema $schema): void
 
             foreach ($data as $row) {
                 if (!empty($row['id'])) {
-                    if ($row['language'] === null) {
+                    if (!array_key_exists('language',$row) || $row['language'] === null) {
```

Comment:
I would use `isset`, as it checks if the key exists and is not `null`.

```suggestion
                    if (!isset($row['language'])) {
```